src/test/java/api/queue/RequestQueueResourceTest.java

```diff
@@ -223,6 +231,101 @@ void refuseAttemptToMoveRequestBeingFulfilledFromOneOfTheTopPositionsWhenTlrEnab
       is("Requests can not be displaced from top positions when fulfillment begun."));
   }
 
+  @Test
+  void shouldGetRequestQueueForItemSuccessfully() {
+    UUID facultyGroupId = patronGroupsFixture.faculty().getId();
+    UUID staffGroupId = patronGroupsFixture.staff().getId();
+    UUID isbnIdentifierId = identifierTypesFixture.isbn().getId();
+    String isbnValue = "9780866989732";
+    item = itemsFixture
+      .basedUponSmallAngryPlanet(
+      identity(),
+      instanceBuilder -> instanceBuilder.addIdentifier(isbnIdentifierId, isbnValue).withId(instanceId),
+      itemBuilder -> itemBuilder
+        .withCallNumber("itCn", "itCnPrefix", "itCnSuffix")
+        .withEnumeration("enumeration1")
+        .withChronology("chronology")
+        .withVolume("vol.1")
+        .withCopyNumber("1"));
```

Comment:
It looks like we're not checking these values anywhere. Do we need them?

Reply:
fixed